plugins/feeds/public/cisa_kev.py

```diff
@@ -0,0 +1,137 @@
+        description += f"* **Vector:** {cvss_data.get('vectorString')}\n"
+        description += f"* **Impact Score:** {cve.get('impactScore')}\n"
+        description += f"* **Exploitability Score:** {cve.get('exploitabilityScore')}\n"
+        if cvss_version == 2:
+            severity = cvss_metric.get('baseSeverity', 'none').lower()
+            description += f"* **Access Vector (AV):** {cvss_data.get('accessVector').capitalize()}\n"
+            description += f"* **Access Complexity (C):**: {cvss_data.get('accessComplexity').capitalize()}\n"
+            description += f"* **Authentication (AU):** {cvss_data.get('authentication').capitalize()}\n"
+            description += f"* **Confidentiality (C):**: {cvss_data.get('confidentialityImpact').capitalize()}\n"
+            description += f"* **Integrigty (I):**: {cvss_data.get('integrityImpact').capitalize()}\n"
+            description += f"* **Availability (A):** {cvss_data.get('availabilityImpact').capitalize()}\n\n"
+        else:
+            severity = cvss_data.get('baseSeverity', 'none').lower()
+            description += f"* **Attack Vector (AV):** {cvss_data.get('attackVector').capitalize()}\n"
+            description += f"* **Attack Complexity (AC):** {cvss_data.get('attackComplexity').capitalize()}\n"
+            description += f"* **Privileges Required (PR):** {cvss_data.get('privilegesRequired').capitalize()}\n"
+            description += f"* **User Interaction (UI):** {cvss_data.get('userInteraction').capitalize()}\n"
+            description += f"* **Scope (S):** {cvss_data.get('scope').capitalize()}\n"
+            description += f"* **Confidentiality (C):** {cvss_data.get('confidentialityImpact').capitalize()}\n"
+            description += f"* **Integrety (I):** {cvss_data.get('integrityImpact').capitalize()}\n"
+            description += f"* **Availability (A):** {cvss_data.get('availabilityImpact').capitalize()}\n\n"
+        description += "#### References\n\n"
+        for reference in cve.get('references', []):
+            tags = " ".join([f"`{tag}`" for tag in reference.get('tags', [])])
+            description += f"* {reference.get('url')} {tags}\n"
+        return base_score, severity, description
+
+
+    def analyze_entry(self, entry: dict, cve_details: dict):
+        """Analyzes an entry as specified in cisa kev json."""
+        try:
+            created = datetime.strptime(entry["dateAdded"], "%Y-%m-%d")
+        except ValueError as error:
+            logging.error("Error parsing kev %s: %s", entry["dateAdded"], error)
+            return
+
+        cve_id = entry.get('cveID')
+        reference = f'https://nvd.nist.gov/vuln/detail/{cve_id}'
+        description = "#### Summary\n\n"
+        description = entry.get("shortDescription") + "\n\n"
+        description += f"* Added to KEV: {entry['dateAdded']}\n"
+        description += f"* Vendor/Project: {entry.get('vendorProject')}\n"
+        description += f"* Product: {entry.get('product')}\n"
```

Comment:
if `get` here fails, we're left with `None`. Better add a default return value like "N/A" ?

Reply:
Fixed!